web/pgadmin/static/js/components/PgTree/FileTreeItem/DoubleClickHandler.jsx

```diff
@@ -15,7 +15,7 @@ import CustomPropTypes from '../../../../js/custom_prop_types';
 export default function DoubleClickHandler({onSingleClick, onDoubleClick, children}){
   const onClick = useSingleAndDoubleClick(onSingleClick, onDoubleClick) ;
   return(
-    <div onClick={(e)=>onClick(e)}>
+    <div onClick={(e)=>onClick(e)} onKeyDown={(e)=>onClick(e)}>
```

Comment:
There is no filter on which key in onKeyDown. Generally, enter key means a click.